pages/index/index_layout.py

```diff
@@ -152,6 +152,14 @@
                     is_open=False,
                     color="info",
                 ),
+                dbc.Alert(
+                    children="List of repos",
+                    id="repo-list-alert",
+                    dismissable=True,
+                    fade=True,
+                    is_open=False,
+                    color="light",
```

Comment:
style={"overflow-y": "scroll", "max-height": "440px"}

Makes the window scrollable and limits total height if there are a lot of entries. Window will be small if there are few.

Reply:
@JamesKunstle : I think @cdolfi made the changes you requested ... Looking to get this tested today If I can.

Reply:
I don't see the requested changes on the branch. I can merge and make my own PR for them though